titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/RDSSSLSocketFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.jooqflyway.jobactivity;
+
+import java.io.InputStream;
+import java.security.KeyStore;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.postgresql.ssl.WrappedFactory;
+
+public class RDSSSLSocketFactory extends WrappedFactory {
+    public RDSSSLSocketFactory() {
+        try {
+            KeyStore trustStore = KeyStore.getInstance(KeyStore.getDefaultType());
```

Comment:
This code is duplicated (we should call into RdsUtil added below).

Reply:
fixed it.